tests/modules/icgc-argo-workflows/payload/qcmetrics/test.yml

```diff
@@ -0,0 +1,8 @@
+- name: "payload qcmetrics"
+  command: nextflow run ./tests/modules/icgc-argo-workflows/payload/qcmetrics -entry test_payload_qcmetrics -c ./tests/config/nextflow.config -c ./tests/modules/icgc-argo-workflows/payload/qcmetrics/nextflow.config -profile docker
```

Comment:
Include a more complicated example involving `fastqc`,`multiqc` and `cutadapt` files to test all functions

Reply:
add test data for `fastqc` and `multiqc`.